src/components/Flyout/Flyout.tsx

```diff
@@ -28,14 +28,14 @@ export const FLYOUT_DIVIDER_COLOR = '#eaebeb';
 export const FLYOUT_DIVIDER_HEIGHT = '10px';
 
 export enum FlyoutPlacement {
-    Top = "top",
-    Bottom = "bottom",
-    TopLeft = "top left",
-    BottomLeft = "bottom left",
-    TopRight = "top right",
-    BottomRight = "bottom right",
-    Right = "right",
-    Left = "left",
+    Top = 'top',
```

Comment:
prettier config changed automatically

Reply:
Yes, we just updated the rules from double quote to single.

Reply:
Yes, we just updated the rules from double quote to single.